async-nats/src/lib.rs

```diff
@@ -877,12 +967,12 @@ pub async fn connect_with_options<A: ToServerAddrs>(
     };
     client
         .sender
-        .send(ClientOp::Connect(connect_info))
+        .send(Command::Connect(connect_info))
```

Comment:
Should probably just write directly here, but for another time.

Reply:
First pass for consistency. We can do those changes another time ;).